op-e2e/Makefile

```diff
@@ -1,14 +1,29 @@
+test: pre-test
+	go test -v ./...
+
+cannon-prestate:
+	make -C .. cannon-prestate
+
 devnet-allocs:
-	PYTHONPATH=../bedrock-devnet python3 ../bedrock-devnet/main.py --monorepo-dir=.. --allocs
+	make -C .. devnet-allocs
+
+
+pre-test: pre-test-cannon pre-test-allocs
```

Comment:
I'd suggest we also update the circleci config to run this rather than each individual component. That way circleci will enforce that this stays up to date as dependencies change in the future.

Reply:
just run `make test`? I'd assume that nothing would exist yet, but we could also do `make pre-test` to be extra sure.

Reply:
wait actually, `make pre-test` always gets ran, the only question is if `make cannon-prestate` or `make devnet-allocs` gets ran (via if files exist).

Reply:
ok, so we use a custom runner rather than `make test` so I updated this.